src/components/modal/Modal.js

```diff
@@ -6,11 +6,17 @@ class Modal extends React.Component {
   constructor(props) {
     super(props);
     this.modal = document.createElement('div');
-    document.body.appendChild(this.modal);
+    const { body } = document;
+
+    body.classList.add('overflow-hidden');
+    body.appendChild(this.modal);
```

Comment:
This may be personal preference, but I generally put anything that's a side effect (i.e. non-React code that doesn't affect component state) inside of a React lifecycle method. `componentDidMount` in this case. Both of these complete instantly though, so I don't think it's a big deal.

Reply:
Agreed, please move this code to: `componentDidMount`

Reply:
I just did that. :-)

I've improved my React (and awareness of my code) skill in this very first PR. And I'm helping to build a wonderful platform.

Thank you guys. You rock!